ayu.md

```diff
@@ -0,0 +1,89 @@
+# Ayu Themes
+
+![AyuMirage](images/AyuMirage.png)
+![AyuDark](images/AyuDark.png)
+![AyuLight](images/AyuLight.png)
+
+## Settings
+
+```json
+{
+    "colorScheme": "Ayu Mirage"
+}
+```
+
+## Fonts
+
+- Cascadia Mono
```

Comment:
Hi @botex98, by any changes, don't have the font? There's a `fonts/` folder where we stored all the fonts... this way, anyone can follow the settings and the set up is much more easy.

Reply:
It's really easy to find and they're continously working on it...
Should I add the followin link to the font in the `ayu.md` file?
https://github.com/microsoft/cascadia-code

Reply:
@rjcarneiro I've added the latest release.